geniza/annotations/views.py

```diff
@@ -74,11 +76,24 @@ def parse_annotation_data(request):
         )
     except Footnote.DoesNotExist:
         source = Source.objects.get(pk=source_id)
+        # try to find a corresponding non-digital footnote
+        # (i.e. Translation for Digital Translation, Edition for Digital Edition)
```

Comment:
could you add one more line about the assumption here? am I understanding correctly that we're assuming if there's exactly one non-digital footnote for this source then the digital content is coming from the same location?